src/stat.cc

```diff
@@ -995,6 +995,14 @@ GetAvgStat(Mgr::IntervalActionData& stats, int minutes, int hours)
     stats.swap_files_cleaned = XAVG(swap.files_cleaned);
     stats.aborted_requests = XAVG(aborted_requests);
 
+    if (Config.paranoid_hit_validation.count()) {
+        stats.hitValidationAttempts = XAVG(hitValidation.attempts);
+        stats.hitValidationRefusalsDueToLocking = XAVG(hitValidation.refusalsDueToLocking);
```

Comment:
A similar "instant condition screws up historical stats" concern applies here and in several other areas where `Config.paranoid_hit_validation.count` is checked in stats-related code AFAICT. I am not going to duplicate this comment for each suspect. Please check all of them.

Reply:
Fixed this and other similar cases.